### PR TITLE
Updated test class names to be more consistent

### PR DESCRIPTION
**Pull request by tmahmood-microsoft**

### CODE REVIEW

**Comment by timtay-microsoft** [comment]
There are several namespaces with "namespace Microsoft.Azure.Devices.E2ETests.iothub.service" that should be "namespace Microsoft.Azure.Devices.E2ETests.IotHub.Service". Can you change those while you are at it?

**Review by timtay-microsoft — APPROVED** [review]

File: e2e/test/iothub/messaging/MessageFeedbackE2ETests.cs [hunk 1]
```diff
@@ -16,11 +16,11 @@ namespace Microsoft.Azure.Devices.E2ETests.Messaging
     [TestClass]
     [TestCategory("E2E")]
     [TestCategory("IoTHub")]
-    public class MessageFeedbackE2eTests : E2EMsTestBase
+    public class MessageFeedbackE2ETests : E2EMsTestBase
```
Comment (drwill-ms) [inline e2e/test/iothub/messaging/MessageFeedbackE2ETests.cs:19]: Casing rules would dictate E2e (lowercase e).

**Pull request merged**